src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/PlatformInterop.java

```diff
@@ -64,6 +71,16 @@
 import java.util.List;
 
 public class PlatformInterop {
+    private static final int MAUI_GLIDE_CLEARED_TAG_ID = R.id.maui_glide_cleared_tag;
+    
+    public static void setGlideClearedTag(View view, boolean isCleared) {
+        view.setTag(MAUI_GLIDE_CLEARED_TAG_ID, isCleared);
+    }
```

Comment:
Instead of using `view.setTag()`, could we have a `boolean isCleared` on `PlatformAppCompatImageView` instead? Could it just be a regular field, with get/set methods? I'm not sure how often developers would try to use the tag for other data, but they can.

Reply:
I used the tag simply because I wanted to provide this functionality also on other views.
But now I realize this only makes sense on Glide View Targets which happens to be only the image view.
So I agree with you, I'll move this to a property.

Reply:
On a second thought there was another, more important, reason.
At the moment `ImageHandler` is exposing `ImageView` so customers may have overridden the handler and provide a different platform view (providing their subclass of `ImageView`).
For those customers we still want to fix #14471 while it is acceptable (I guess) to leave them with an eventual flicker.